runtime/src/impls.rs

```diff
@@ -35,6 +36,45 @@ use sp_runtime::{
 	DispatchError, Fixed64,
 };
 
+pub type NegativeImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::NegativeImbalance;
+pub type PositiveImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::PositiveImbalance;
+
+type CennzxSpot<T> = crml_cennzx_spot::Module<T>;
+type Contracts<T> = pallet_contracts::Module<T>;
+type GenericAsset<T> = pallet_generic_asset::Module<T>;
+type Staking<T> = pallet_staking::Module<T>;
+
+pub struct SplitToAllValidators;
+
+/// This handles the ```NegativeImbalance``` created for transaction fee.
+/// The reward is split evenly and distributed to all of the current elected validators.
+/// The remainder from the division are burned.
+impl OnUnbalanced<NegativeImbalance> for SplitToAllValidators {
+	fn on_nonzero_unbalanced(amount: NegativeImbalance) {
+		let validators = Staking::<Runtime>::current_elected();
+		if validators.len() > 0 || amount.peek().is_zero(){
+			// Get a list of elected validators
+			let per_validator_reward: Balance = amount.peek() / (validators.len() as Balance);
+
+			// This tracks the total amount of reward actually handed out. Used to adjust total issurance
+			let mut total_imbalance = PositiveImbalance::zero();
+			let asset_id = amount.asset_id();
+			for v in &validators {
+				let payout = <GenericAsset<Runtime> as MultiCurrencyAccounting>::deposit_creating(
+					&v,
```

Comment:
This should pay to the validator reward destination account: https://github.com/plugblockchain/plug-blockchain/blob/d2319b9c98c209be11544e5b05b4797e670c5be6/frame/staking/src/lib.rs#L696

Reply:
Implemented the required logic